pkg/eventcounter/api/api.go

```diff
@@ -79,16 +81,20 @@ func NewEventCounterService(
 	f featureclient.Client,
 	a accountclient.Client,
 	d ecdruid.Querier,
+	b bqquerier.Client,
+	bigQueryDataSet string,
 	r metrics.Registerer,
 	redis cache.MultiGetDeleteCountCache,
 	l *zap.Logger,
 ) rpc.Service {
 	registerMetrics(r)
 	return &eventCounterService{
-		experimentClient:             e,
-		featureClient:                f,
-		accountClient:                a,
-		druidQuerier:                 d,
+		experimentClient: e,
+		featureClient:    f,
+		accountClient:    a,
+		druidQuerier:     d,
+		// v2ecstorage.NewEventStorage() is executed at here instead of at eventcounter/cmd/server because {module}/cmd/server should not have a dependency to {module}/storage
```

Comment:
Because we are not doing anything special here, I think this comment is unnecessary.
This is the way we implemented all the services.